dm/dm/worker/metrics.go

```diff
@@ -104,7 +105,8 @@ func (s *Server) runBackgroundJob(ctx context.Context) {
 func RegistryMetrics() {
 	registry := prometheus.NewRegistry()
 	registry.MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
-	registry.MustRegister(prometheus.NewGoCollector())
+	registry.MustRegister(prometheus.NewGoCollector(
```

Comment:
Pls modify the   `NewGoCollector()` in  cdc/metrics.go.

Reply:
thanks addressed in https://github.com/pingcap/tiflow/pull/5643/commits/c6ac8c0155eeb449266a354f09006a430a87ae70